src/main/java/org/example/dao/TodoDAO.java

```diff
@@ -34,11 +35,12 @@ public static void createTaskDB(Task task) {
 
     }
 
-    public static void getAllTasksDB() {
+    public static ArrayList<Task> getAllTasksDB() {
```

Comment:
For the next iteration, use an Immutable array here.

Reply:
Perfect,  I will